src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -1021,8 +992,8 @@ public void DbPolicy_ClaimValueTypeParsing(string claimValueType, string claimVa
 
             Mock<HttpContext> context = new();
 
-            //Add identity object to the Mock context object.
-            ClaimsIdentity identity = new(TEST_AUTHENTICATION_TYPE, TEST_CLAIMTYPE_NAME, AuthenticationConfig.ROLE_CLAIM_TYPE);
+            //sAdd identity to the readAction, updateActionext.
```

Comment:
```suggestion
            //Add identity to the request context.
```
looks like find and replace accidental change